app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,60 @@
+<html lang="en">
+<%= render partial: '/layouts/login/devise'%>
```

Comment:
Muốn sử dụng 1 layout khác (thay vì chung layout admin) cho devise sử dụng cách sau https://github.com/heartcombo/devise/wiki/How-To%3A-Create-custom-layouts